chargeio/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ChargeIOSecretKey": "<Your OAuth Secret>",
+    "ChargeIOApiUrl": "https://api.chargeio.com",
+    "ApiVersion": "/v1",
```

Comment:
We shouldn't split ApiVersion out of ApiUrl -- it assumes a URL structure which may not always hold. Also, at this point, we view inclusion of the "/v1" as unfortunate, and are not reproducing with the more recent APIs (since there are better ways to perform REST versioning).